include/databaseFunctions_blog_pic.php

```diff
@@ -152,19 +352,24 @@ function DeletePost($postID){
 			WHERE postID = $postID
 		")->fetch();
 		echo"Post Deleted.<br>";
+		ResetAuto(GetTotalPosts());
 	}
 
-	function isPicture($postID){
-		if(GetPost($postID)['isPic']==1){
-			return true;
-		}
-		return false;
+	function GetPostType($postID){
+		return GetPost($postID)['postType'];
 	}
 
-	function getTotalPosts(){
+	function GetTotalPosts(){
 		$result = dbQuery("
-			SELECT COUNT(postID) AS count
+			SELECT MAX(postID) AS count
```

Comment:
It's not necessarily safe to assume that the max post id is the number of posts. For example, you might delete some posts, or your auto-increment count might get messed up (this happens sometimes if you e.g. migrate to a new database)